bin/Grid_Bathy/hycom_bathy.py

```diff
@@ -34,20 +39,22 @@ def main(infile,blo,bla,shapiro_passes,resolution=None,cutoff=5.) :
    scpy=gfile.read_field("scpy")
    width=numpy.median(scpx)
    logger.info("Grid median resolution:%8.2f km "%(width/1000.))
-
+   # give the bathy directory here
+   #bathyDir="/work/shared/nersc/msc/ModelInput/bathymetry/GEBCO_2014/"
+   bathyDir="/nobackup/prod2/sm_alfal/ModelInput/bathymetry/GEBCO/"
```

Comment:
@AlfaAli I prefer that we keep the project directory path in the github version

Reply:
Do mean we should use this dir.:/cluster/projects/nn2993k/ModelInput/bathymetry/GEBCO_2014/?
or  we may simply use bathyDir="./"

Reply:
We should use this one:  dir.:/cluster/projects/nn2993k/ModelInput/bathymetry/GEBCO_2014/, since most of us is using that